src/storage/core.rs

```diff
@@ -911,7 +911,7 @@ where
                     None
                 }
             })
-            .fold(0, |a, b| a + b)
+            .fold(0, |a, _| a + 1)
```

Comment:
`count`?

Reply:
It was buried under ambigous calls, fixed